Configuration/Skimming/python/autoSkim.py

```diff
@@ -1,20 +1,17 @@
 autoSkim = {
  'BTagMu' : 'LogError+LogErrorMonitor',
- 'JetHT' : 'JetHTJetPlusHOFilter+LogError+LogErrorMonitor',
  'DisplacedJet' : 'EXODisplacedJet+EXODelayedJet+EXODTCluster+EXOCSCCluster+LogError+LogErrorMonitor',
- 'MET' : 'EXOHighMET+EXODelayedJetMET+LogError+LogErrorMonitor',
+ 'JETMET' : 'JetHTJetPlusHOFilter+EXOHighMET+EXODelayedJetMET+LogError+LogErrorMonitor',
```

Comment:
The name of the dataset is `JetMET` (not `JETMET`). Does this matter here, or is this case-insensitive?

Reply:
@missirol  thanks for the reminder, it is fixed now.